core/evalir.ml

```diff
@@ -275,9 +275,12 @@ struct
               | `ClientPid (client_id, process_id) ->
                   Mailbox.send_client_message msg client_id process_id
            with
-                 UnknownProcessID _ ->
+                 UnknownProcessID id ->
                    (* FIXME: printing out the message might be more useful. *)
```

Comment:
Do we still need that FIXME comment?

I am also surprised to see an exception being replaced with a debug message. But perhaps that's fine - I don't really understand what's going on here.

Reply:
Oops, the FIXME can go.

I think the FIXME was saying that a failed message send to a mailbox (rather than a session channel) needn't be fatal, along with requiring a bit more information.